library/std/src/ffi/os_str.rs

```diff
@@ -32,7 +32,7 @@ use crate::{cmp, fmt, slice};
 /// `OsString` and [`OsStr`] bridge this gap by simultaneously representing Rust
 /// and platform-native string values, and in particular allowing a Rust string
 /// to be converted into an "OS" string with no cost if possible. A consequence
-/// of this is that `OsString` instances are *not* `NUL` terminated; in order
+/// of this is that `OsString` instances are *not* `NULL` terminated; in order
```

Comment:
`NUL` is the correct spelling. [Wikipedia: Null character](<https://en.wikipedia.org/w/index.php?title=Null_character&oldid=1259227006>).